api-specificatie/openapi.yaml

```diff
@@ -110,7 +110,7 @@ paths:
           $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/default'
   # JBO: Het ontsluiten van een adres op basis van de unieke adres-id wordt gebruikelijk gedaan met een path-parameter. Dat impliceert wel een unieke identificatie en is dus geen zoeken meer maar raadplegen. Dat klopt met de response definitie waarin geen array, maar een enkel adres wordt geretourneerd
   #JBO: Krijgen alle adressen nu een eigen adres-id of gaan we de nummeraanduiding-id gebruiken als adres-id ?
-  /adressen/{key}:
+  /adressen/{nummeraanduidingidentificatie}:
```

Comment:
Volgens de goals canvas, moet hier met een adres string of een nummeraanduiding gezocht/bevraagd kunnen worden. Het lijkt mij daarom noodzakelijk om een /adressen endpoint te hebben met 2 optionele query parameters (adres string en NUM ID). Waarbij er in de implementatie dan logica geïmplementeerd moet worden om te controleren dat er één van de 2 parameters is opgegeven en niet 0 of 2. Een adres is geen bestaand object in de BAG en heeft daarom geen adres ID, in plaats daarvan wordt het nummeraanduiding ID gebruikt.

Reply:
Los van de discussie of het zoeken op een adresstring hier gewenst is (zie #44) Kan dit anders opgelost worden. Het nummeraanduidingID is een unieke identificatie van de resource adres (immers 1 adres correspondeert met 1 nummeraanduiding) . Dat kan je dus realiseren door een de huidige implementatie van /adressen/{nummeraanduidingidentificatie}

Als de query-parameter adresstring nodig blijkt te zijn definieren we een endpoint /adressen met als query-parameter de adresstring.